README.md

```diff
@@ -77,6 +83,9 @@ and modify the `system-reserved` line under `kubeletExtraArgs`:
 ./deploy-konflux.sh
 ```
 
+**Note:** If you get the following error: `error: json: unknown field "replacements"`,
+you will need to update your `kubectl`.
+
```

Comment:
Should I move this one to the troubleshooting section as well? @yftacherzog @avi-biton

Reply:
Sounds good!